src/equilibrate_utils.jl

```diff
@@ -0,0 +1,726 @@
+#Collection of functions relating to finding fixed points of reaction networks, as well as the creation of bifurcation diagrams.
+
+### Functions used during the construction of the reaction network. ###
+
+#Equilibrate content structure, with all the stuff required for homotopy continuation base equilibration calculations.
+mutable struct EquilibrateContent
+    make_polynomial::Function
+    constraints::Vector{Polynomial{true,Float64}}
+    homotopy_continuation_templates::Vector{NamedTuple{(:p, :sol),Tuple{Vector{Complex{Float64}},Vector{Vector{Complex{Float64}}}}}}
+    equilibrium_polynomial::Union{Vector{Polynomial{true,Float64}},Nothing}
+    is_polynomial_system::Bool
+    polyvars::NamedTuple{(:x, :t, :p),Tuple{Vector{PolyVar{true}},PolyVar{true},Vector{PolyVar{true}}}}
```

Comment:
x?

Reply:
I have removed all mentions of x and replaced them so that only u should be used.